client/src/Styles/Leaderboard.css

```diff
@@ -0,0 +1,41 @@
+body {
```

Comment:
I would avoid using body, it causes issues in my current rewrite I'm working on where it kept changing background colours or rendering unexpected styles. Same goes for using the same style names across pages for some reason.

Wrapping around a div should fix that

Reply:
Added a class to the div that was already wrapping the leaderboard component.